mvp-spec/ad-server-implementation.md

```diff
@@ -214,7 +214,7 @@ we name `data` in the following example:
 <!--partial-end-->
 
 Similar to the Seed, each Transmission Request contains a signature for 
-audit purposes. It used the data of the Transaction Request and the domain
+audit purposes. It used the data of the Transmission Request and the domain
```

Comment:
Typo: It *uses*

missing info: it also uses the seed signature

Reply:
Done